### ISSUE TITLE
Marketing Video - Maestro Test

### ISSUE THREAD

**Issue by GantMan** [infinitered/ChainReactApp2023#163]
Show AI tab and ask the following questions (super fast typing with maestro)

Ask the following questions:

* Who are the speakers from Meta?
* When are their talks?
* Where is the conference?
* What are some good places to eat near there?

Supply the video for us to share.

**Comment by mazenchami** [comment]
@GantMan v1 of the Maestro Flow:

https://user-images.githubusercontent.com/9324607/234087641-eda06a8e-ae50-464a-942b-0079458b699c.mp4

**Comment by mazenchami** [comment]
ok, let me know what you think about this:

https://user-images.githubusercontent.com/9324607/234088515-5995ecaa-1fda-4caa-9a77-4bd80ec0cfcc.mp4

**Comment by mazenchami** [comment]
cc: @JenFucci @justinhuskey

**Comment by jamonholmgren** [comment]
Very cool video!

**Comment by GantMan** [comment]
love it.  Will the context be updated to suggest food from our maps?

**Comment by GantMan** [comment]
For posterity
![ai_in_action](https://user-images.githubusercontent.com/997157/234109730-edfb0ffd-d3e9-4e43-ba23-871aa57848a2.gif)

**Comment by mazenchami** [comment]
@GantMan yes it will. I'm working on a PR for the prompt now based off of what we discussed on Friday. i'll update the video once i get it merged
